.gitignore

```diff
@@ -0,0 +1 @@
+*.tar
```

Comment:
Is this really needed? I could not find any generation or anything else in the code related to tar file. Is it something specific to pgbouncer?

Reply:
This was added because I was storing containers locally as .tar files to import into microk8s container registry, but it's not really necessary for the repo so I've removed it :)